google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/spi/v1/Subscriber.java

```diff
@@ -244,10 +243,17 @@ public void close() {
       throw new IllegalStateException(e);
     }
 
-    // Streaming pull is not enabled on the service yet.
-    // startStreamingConnections();
-    startPollingConnections();
-    notifyStarted();
+    new Thread(new Runnable() {
+      @Override
+      public void run() {
+        try {
+          startPollingConnections();
+          notifyStarted();
+        } catch (Throwable t) {
+          notifyFailed(t);
+        }
+      }
+    }).start();
```

Comment:
IMHO it be better if we use the "system/library" executor for this. But since starting subscribers should be an operation that is not often executed, I'm fine if we spawn a separate thread on every execution.